racetrack_commons/racetrack_commons/auth/scope.py

```diff
@@ -2,10 +2,10 @@
 
 
 class AuthScope(Enum):
-    READ_FATMAN = 'read_fatman'  # list fatman, check fatman details
-    DEPLOY_FATMAN = 'deploy_fatman'  # deploy fatman in a particular family, redeploy fatman
-    DEPLOY_NEW_FAMILY = 'deploy_new_family'  # deploy new fatman family
-    DELETE_FATMAN = 'delete_fatman'  # move to trash, dismantle from a cluster
-    CALL_FATMAN = 'call_fatman'  # call fatman endpoints
+    READ_JOB = 'read_job'  # list job, check job details
+    DEPLOY_JOB = 'deploy_job'  # deploy job in a particular family, redeploy job
+    DEPLOY_NEW_FAMILY = 'deploy_new_family'  # deploy new job family
+    DELETE_JOB = 'delete_job'  # move to trash, dismantle from a cluster
+    CALL_JOB = 'call_job'  # call job endpoints
```

Comment:
Let's keep old scopes for a while as these values are kept in a database and need to be migrated separately
```suggestion
    READ_JOB = 'read_fatman'  # list job, check job details
    DEPLOY_JOB = 'deploy_fatman'  # deploy job in a particular family, redeploy job
    DEPLOY_NEW_FAMILY = 'deploy_new_family'  # deploy new job family
    DELETE_JOB = 'delete_fatman'  # move to trash, dismantle from a cluster
    CALL_JOB = 'call_fatman'  # call job endpoints
```